scss/mixins/_utilities.scss

```diff
@@ -44,6 +47,16 @@
           #{$property}: $value if($enable-important-utilities, !important, null);
         }
       }
+
+      @if $state {
```

Comment:
Can we avoid the if test here if we use `()` as fallback in L24?

Reply:
This is a great idea. Thanks. I Updated PR